package.json

```diff
@@ -93,95 +93,96 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "7.0.0-beta.44",
-    "@babel/plugin-proposal-async-generator-functions": "7.0.0-beta.44",
-    "@babel/plugin-proposal-object-rest-spread": "7.0.0-beta.44",
-    "@babel/plugin-transform-flow-comments": "7.0.0-beta.44",
-    "@babel/plugin-transform-runtime": "7.0.0-beta.44",
-    "@babel/preset-env": "7.0.0-beta.44",
-    "@babel/runtime": "7.0.0-beta.44",
+    "@babel/core": "7.0.1",
+    "@babel/plugin-proposal-async-generator-functions": "7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0",
+    "@babel/plugin-transform-flow-comments": "7.0.0",
+    "@babel/plugin-transform-runtime": "7.0.0",
+    "@babel/preset-env": "7.0.0",
+    "@babel/runtime": "7.0.0",
     "@zeit/dockerignore": "0.0.3",
     "@zeit/source-map-support": "0.6.2",
-    "ajv": "6.4.0",
+    "ajv": "6.5.3",
     "alpha-sort": "2.0.1",
-    "ansi-escapes": "3.0.0",
+    "ansi-escapes": "3.1.0",
     "ansi-regex": "3.0.0",
-    "archiver": "2.0.3",
+    "archiver": "3.0.0",
     "arg": "2.0.0",
     "arr-flatten": "1.1.0",
-    "async-retry": "1.1.3",
-    "async-sema": "1.2.0",
+    "async-retry": "1.2.3",
+    "async-sema": "2.1.2",
     "ava": "0.25.0",
-    "aws-sdk": "2.124.0",
-    "babel-eslint": "8.2.2",
+    "aws-sdk": "2.315.0",
```

Comment:
Looks like all deps were upgraded instead of just the ones related to webpack

Reply:
that's bad?

Reply:
Not necessarily, but I'd separate upgrading dependencies from implementing webpack 4, as both can be fragile changes.

Reply:
Check each dependency you were updating to see if there are important changes, but most solve problems from previous versions : )

Reply:
What do you recommend me to do ? :(